cdc/sink/mq.go

```diff
@@ -212,8 +219,14 @@ func (k *mqSink) run(ctx context.Context) error {
 		}
 
 		// wait mq producer send message successfully
-		for sinkCheckpoint.index > k.mqProducer.MaxSuccessesIndex() {
-			time.Sleep(20 * time.Millisecond)
+		err := retry.Run(func() error {
```

Comment:
the `DefaultInitialInterval` for retry is 500ms. 🤔️
Seems our capsulation for retry shadows the retry config

Reply:
let me expose this config in next pr

Reply:
see #378

Reply:
fine